Makefile

```diff
@@ -16,28 +16,34 @@ new_env: clean_env pre_env env
 clean:
 	rm -rf build dist target_python_sdk.egg-info
 
-# Install dependencies - pip update sometimes required for python 2.7
+# Install dependencies - older versions of python need to bootstrap pip for CI workflow
 install: env
-	curl https://bootstrap.pypa.io/get-pip.py | python
-	pip install --upgrade pip "setuptools>=38.3.0,<=44.1.x" wheel pep517 pylint twine bump2version coverage coveralls
-	pip install -r requirements.txt
-	pip install -r test-requirements.txt
+ifneq (,$(filter $(PYTHON_VERSION),2.7 3.4 3.5))
```

Comment:
I'm surprised the github workflow image doesn't already include a proper pip.

Reply:
It includes the latest version, which doesn't work for older python builds